cms/static/js/views/course_video_settings.js

```diff
@@ -0,0 +1,545 @@
+                            {
+                                languageCode: selectedLanguage,
+                                text: gettext('Remove'),
+                                srText: gettext('Press Remove to remove language')
+                            }
+                        )
+                    )
+                );
+
+                // Add a new language menu
+                this.addLanguageMenu();
+            } else {
+                this.addErrorState($languagesEl);
+            }
+        },
+
+        languageRemoved: function(event) {
+            var selectedLanguage = $(event.target).data('language-code');
+            $(event.target.parentElement).parent().remove();
+            this.selectedLanguages.pop(selectedLanguage);
+        },
+
+        populateProvider: function() {
+            var self = this,
+                providerPlan = self.getProviderPlan(),
+                $providerEl = self.$el.find('.transcript-provider-group');
+
+            if (providerPlan) {
+                $providerEl.empty();
+                HtmlUtils.setHtml(
+                    $providerEl,
+                    HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML('<input type="radio" id="transcript-provider-none" name="transcript-provider" value="" {checked}/><label for="transcript-provider-none">{text}</label>'),
+                        {
+                            text: gettext('None'),
+                            checked: self.selectedProvider === '' ? 'checked' : ''
+                        }
+                    )
+                );
+
+                _.each(providerPlan, function (providerObject, key) {
+                    var checked = self.selectedProvider === key ? 'checked' : '';
+                    HtmlUtils.append(
+                        $providerEl,
+                        HtmlUtils.interpolateHtml(
+                            HtmlUtils.HTML('<input type="radio" id="transcript-provider-{value}" name="transcript-provider" value="{value}" {checked}/><label for="transcript-provider-{value}">{text}'),
+                            {
+                                text: providerObject.display_name,
+                                value: key,
+                                checked: checked
+                            }
+                        )
+                    )
+                });
+            }
+        },
+
+        populateTurnaround: function() {
+            var self = this,
+                turnaroundPlan = self.getTurnaroundPlan(),
+                $turnaroundContainer = self.$el.find('.transcript-turnaround-wrapper'),
+                $turnaround = self.$el.find('#transcript-turnaround');
+
+            // Clear error state if present any.
+            this.clearPreferanceErrorState($turnaroundContainer);
+
+            if(self.selectedProvider && turnaroundPlan) {
```

Comment:
`if(turnaroundPlan)` is enough as `turnaroundPlan` is already checking `self.selectedProvider`